src/pages/Clients/Eth1/Besu.tsx

```diff
@@ -14,57 +14,62 @@ export const BesuDetails = () => (
       Hyperledger Besu
     </SectionTitle>
     <Text className="mt10">
-      An Ethereum client designed to be enterprise-friendly for both public and
-      private, permissioned network use cases.
+      <FormattedMessage
+        defaultMessage="An Ethereum client designed to be enterprise-friendly for
+          both public and private, permissioned network use cases."
+      />
     </Text>
     <SectionTitle level={2} className="mb5">
-      Language information
+      <FormattedMessage defaultMessage="Language information" />
     </SectionTitle>
     <Text className="mt10">
-      Besu is written in Java and released under the Apache 2.0 Licence.
+      <FormattedMessage defaultMessage="Besu is written in Java and released under the Apache 2.0 Licence." />
     </Text>
     <section>
       <SectionTitle level={2} className="mb5">
-        Installation
+        <FormattedMessage defaultMessage="Installation" />
       </SectionTitle>
       <Text className="mt10">
-        The Besu documentation explains how to download and install the client.
+        <FormattedMessage defaultMessage="The Besu documentation explains how to download and install the client." />
       </Text>
       <Link
         className="mt10"
         primary
         to="https://besu.hyperledger.org/en/stable/HowTo/Get-Started/Install-Binaries/"
       >
-        Besu installation documentation
+        <FormattedMessage defaultMessage="Besu installation documentation" />
       </Link>
     </section>
     <section>
       <SectionTitle level={2} className="mb5">
-        Setup
+        <FormattedMessage defaultMessage="Setup" />
       </SectionTitle>
       <Text className="mb20 mt10">
-        Make sure you do the following to get your Eth1 client working properly.
+        <FormattedMessage defaultMessage="Make sure you do the following to get your Eth1 client working properly." />
       </Text>
       {!IS_MAINNET && (
         <>
           <Heading level={3} className="mt20">
-            Testing on Goerli
+            <FormattedMessage defaultMessage="Testing on Goerli" />
           </Heading>
           <Text className="mt10">
             <FormattedMessage
               defaultMessage="Use {goerli} to sync the Goerli testnet."
               values={{
                 goerli: <Code className="mt20">--network=goerli</Code>,
               }}
+              description="{goerli} shows '--network=goerli' terminal command"
             />
           </Text>
           <Link
             primary
             to="https://besu.hyperledger.org/en/stable/HowTo/Get-Started/Starting-node/#run-a-node-on-goerli-testnet"
             className="mt20"
           >
-            {' '}
-            Besu on Goerli documentation
+            <FormattedMessage
+              defaultMessage="Besu on Goerli documentation"
```

Comment:
I'm going to approve & merge but CC @ryancreatescopy this language we may want to rework so it's clearer for translators.